piksi_tools/console/settings.yaml

```diff
@@ -1701,6 +1701,20 @@
   Description: Enable the almanac-based acquisition.
   Notes:
 
+- group: ethernet
+  name: mode
+  expert: false
+  type: boolean
+  enumerated possible values: True,False
+  units: N/A
+  default value: 'False'
+  readonly: false
+  Description: Ethernet configuration locked toggle.
+  Notes: |
+    "Config" IP configuration can be changed freely, but no change is made on the device.
```

Comment:
Append "Changing setting to Active will refresh ethernet connection with current values."

Reply:
I can't determine how this setting works from the description.  If in "active" mode, does the device behave as it used to on prior releases?

Reply:
Underlying behavior is that we want 'active' to denote currently active settings and delay application of settings changes to individual settings until it is desired to do so. The solution is to change to 'config' mode which allows you to change the setting without them taking effect. Switching back to 'active' will apply the settings and re-up the interface in one go